source/utilities.cc

```diff
@@ -376,7 +385,7 @@ namespace aspect
     {
       std::ifstream in(filename.c_str(), std::ios::in);
       AssertThrow (in,
-                   ExcMessage (std::string("Could not open data file <"
+                   ExcMessage (std::string("Couldn't open data file <"
```

Comment:
I think I've converted a number of places to more formally use "could not" instead of "couldn't".